onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -713,6 +712,25 @@ void QnnBackendManager::Split(std::vector<std::string>& split_string,
   }
 }
 
+Status QnnBackendManager::DestroyHTPPowerConfigID() {
```

Comment:
>DestroyHTPPowerConfigID [](start = 26, length = 23)

Where does this get called?

Reply:
ah forgot. I added it to the ReleaseResources() method.